src/meshapi/views/map.py

```diff
@@ -16,6 +20,14 @@
     MapDataSectorSerializer,
 )
 
+LINKNYC_KIOSK_DATA_URL = "https://data.cityofnewyork.us/resource/s4kf-3yrf.json?$limit=100000"
```

Comment:
Thought: Should we start an `apis.py` where we keep all of these API consts? That way they're in one place so there's no guessing about what we depend on externally.

Reply:
I can do that, but as a separate PR. This one is big already

Reply:
https://github.com/nycmeshnet/meshdb/issues/453